doc/tutorial/getting-started/index.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: doc
+title: Getting Started with Scala.js, Vite, Laminar and ScalablyTyped
+---
+
+This series of tutorials teaches you how to use Scala.js together with modern development tools.
+
+If you have time, reading and applying them in order will give you more in-depth knowledge about the development environment.
+
+If you are in a hurry, you can skip the ones you are not interested in.
+Each tutorial starts with a link to repo that you can clone to get off the ground.
+
+In any case, make sure that you have the Prerequisites listed below covered.
```

Comment:
I find it harder to read that `# Prerequisites` does not directly follow this sentence